Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -284,7 +284,21 @@ MiG_29A:defineToggleSwitch("WEAPONS_CONTROL_BURST_MODE_SWITCH", devices.WP, 3012
 MiG_29A:definePotentiometer("WEAPONS_CONTROL_WING_SPAN_KNOB", devices.INPUT_PANEL, 3001, 520, { 0, 1 }, WEAPONS_CONTROL, "Target Wing Span Knob")
 MiG_29A:defineMultipositionSwitch("WEAPONS_CONTROL_WCS_MODES_SELECTOR", devices.INPUT_PANEL, 3004, 523, 8, 0.1, WEAPONS_CONTROL, "WCS Modes Selector Knob (TOSS/NAV/RAD/IR/CC/HELM/OPT/BS)")
 
--- AFCS Autopilot control pannel
+-- AFCS Autopilot control panel
+local AUTOPILOT = "AFCS Autopilot Control Panel"
+
+MiG_29A:definePushButton("AUTOPILOT_DAMPER_BUTTON", devices.AFCS_INTERFACE, 3001, 82, AUTOPILOT, "Autopilot Damper Button")
+MiG_29A:defineIndicatorLight("AUTOPILOT_DAMPER_LIGHT", 83, AUTOPILOT, "Autopilot Damper Light")
```

Comment:
These indicator lights should be defined as floats as they are 0-1. `defineIndicatorLight` is a binary on/off output. I guess a lot of the indicator lights used to just be on or off - but these are dimmable, so we should use `defineFloat` I believe.

Reply:
Got it! Changed it